macros/unpack/get_metrics.sql

```diff
@@ -32,7 +19,26 @@
             {%- else -%}
                 ''
             {% endif -%}
-        {%- endset -%}
+          {% endset %}
+
+          {%- set values_line = 
```

Comment:
any reason to split these set blocks apart?

Reply:
Actually I can't get them together. I don't think I can use `if`s and `for`s within the variable definition itself.